src/operator/activation-inl.h

```diff
@@ -24,7 +24,7 @@ namespace op {
 namespace activation {
 enum ActivationOpInputs {kData};
 enum ActivationOpOutputs {kOut};
-enum ActivationOpType {kReLU, kSigmoid, kTanh};
+enum ActivationOpType {kReLU, kSigmoid, kTanh, kSoftplus};
```

Comment:
Thanks for pointing this out.  Agreed, I have changed the name to `SoftReLU` and `softrelu`.